### ISSUE TITLE
TLOZ: A Link Between Worlds freezes upon creating a new game

### ISSUE THREAD

**Issue by alexis23ange** [citra-emu/citra#1513]
So when you try and start a new game in a A Link between worlds and click the Begin Button the game fails to start as it should instead it freezes all controls and then this shows in the log now lets try and figure out this issue and not dismiss it.

```
[ 76.395101] Service core\hle\service\service.cpp:Service::Interface::SyncRequest:68: unknown / unimplemented function 'ControlArchive': port=fs:USER, cmd_buff[1]=0x7, cmd_buff[2]=0x0, cmd_buff[3]=0x0, cmd_buff[4]=0x1, cmd_buff[5]=0x1, cmd_buff[6]=0x1A, cmd_buff[7]=0x162F4D60, cmd_buff[8]=0x1C, cmd_buff[9]=0x162F4D64
```

**Comment by yuriks** [comment]
That's much better, but really, try to tune down the snark.

**Comment by alexis23ange** [comment]
no snark you just have a attitude and it doesnt help with progress trying being a little more friendly and less rude

**Comment by Kloen** [comment]
@alexis23ange And a issue without any usefull information it's useless... read this please: http://www.chiark.greenend.org.uk/~sgtatham/bugs.html

BTW. Don't argue, just be smart people, if you want devs to help you, then you should help then by providing usefull information. A lonely line of the log, isn't much in fact.

**Comment by KillzXGaming** [comment]
So this may fix issue
https://github.com/mailwl/citra/blob/ctrl_archive/src/core/hle/service/fs/fs_user.cpp#L407

However in order to boot in game, you need 
https://github.com/citra-emu/citra/pull/1302

**Comment by lioncash** [comment]
@alexis23ange I love ironic posts

**Comment by KillzXGaming** [comment]
Can confirm Sonic Lost Worlds crashes once you go in-game and gives the same error. 
![image](https://cloud.githubusercontent.com/assets/13475262/13863607/4b5aacae-ec71-11e5-8d0e-7a8faeb26f4d.png)

**Comment by alexis23ange** [comment]
they get mad when they don't have enough info yet i give the info the log has haha maybe you guys need to make a better log system for the emulator and stop blaming the users and be smart about it

**Comment by KillzXGaming** [comment]
For future use, please make a proper title of the issue and maybe a log file.
To do this
right click citra folder, open with cmd
paste Citra-qt 1> console.log 2>&1
Use pastebin and paste consle.log

Here i created one with latest master:
[http://pastebin.com/iQ7SXYFj](LOG)

**Comment by Subv** [comment]
This was fixed by #2260
